src/screens/TokenSwap.js

```diff
@@ -204,13 +218,26 @@ const TokenSwap = () => {
     return t`Balance: ${amount}`;
   };
 
-  const onReviewButtonPress = () => {
+  const getAmountWithSlippage = (amount, direction) => {
+    return calcAmountWithSlippage(direction, amount, TOKEN_SWAP_SLIPPAGE);
+  };
+
+  const getAmountString = (amount, token) => {
+    return `${renderValue(amount, false)} ${token.symbol}`;
+  };
+
+  const getConversionRate = (swapQuote) => {
+    if (!swapQuote) {
+      return null;
+    }
+    return `${getAmountString(100*Number(swapQuote.amount_out)/Number(swapQuote.amount_in), outputToken)} = ${getAmountString(100, inputToken)}`;
```

Comment:
question(non-blocking): Why multiply by 100? Won't `renderValue()` deal with the decimal cases already?